src/containers/UserActionsBox/UserActionsBox.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+import '../../styles/UserActionsBox.scss'
+import Delete from '../../components/Delete/Delete'
+
+interface UserActonsBoxProps {
+  delete: (id: number) => void;
+  id: number;
+  update: () => void;
+  updateDeleteStatus: () => void;
+}
+
+const UserActionsBox: React.FC<UserActonsBoxProps> = (props) => {
+
+  const handleDeleteClick = (event: React.FormEvent) => {
+    props.delete(props.id)
+    props.update()
+    props.updateDeleteStatus()
+  }
+
+  return (
+    <div className='UserActionsBox--container'>
+      <Delete 
```

Comment:
So this Delete component we would conditionally render then based on if the user's id matches?